src/data/create_segments.py

```diff
@@ -273,12 +273,22 @@ def add_point_based_features(non_inters, inters, jsonfile,
         near = feature.near_id
         feat_type = feature.properties['feature']
 
+        if 'feat_agg' in feature.properties and 'value' in feature.properties:
+            feat_agg_type = feature.properties['feat_agg']
+        else:
+            feat_agg_type = 'default'
+
         if near:
-            if str(near) not in matches:
-                matches[str(near)] = {}
-            if feat_type not in matches[str(near)]:
-                matches[str(near)][feat_type] = 0
-            matches[str(near)][feat_type] += 1
+            if feat_agg_type == 'latest':
+                if str(near) not in matches:
+                    matches[str(near)] = {}
+                matches[str(near)][feat_type] = feature.properties['value']
```

Comment:
This looks like it's just going to set the value of the feature to the last value it sees as it iterates through the list. The list isn't guaranteed to be in date order though, so I don't think this will get the latest value. You could store the date and value in a list here, and then use the latest value when you add the features to their associated intersection and non-intersection segments below.